### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by JaredT6694** [semaphoreui/semaphore#2003] [Ideas]
I would like to request a feature add, instead of hard-coding the 1001 UID, could the option to pass UID/GID in from the docker run / docker compose be enabled?

**Comment by tboerger** [comment]
Currently there is no fix for customizing the uid/gid, it is hardcoded. If you are not able to give the host folder to uid 1001 you could at least try some docker plugin like https://github.com/clecherbauer/docker-volume-bindfs where you can use bindfs to map from host uids to container uids.
